tests/foreman/cli/test_organization.py

```diff
@@ -55,630 +56,601 @@ def valid_labels_list():
         gen_string('alpha'),
         gen_string('alphanumeric'),
         gen_string('numeric'),
-        '{}-{}'.format(gen_string('alpha', 5), gen_string('alpha', 5)),
-        '{}_{}'.format(gen_string('alpha', 5), gen_string('alpha', 5)),
+        f"{gen_string('alpha', 5)}-{gen_string('alpha', 5)}",
+        f"{gen_string('alpha', 5)}-{gen_string('alpha', 5)}",
```

Comment:
I think the change from "_" to "-" is oversight here. Please fix this. But everything else LGTM.

Reply:
Fixed.